web/source/build.sh

```diff
@@ -85,16 +85,18 @@ minifycmd="$JAVA -jar $minifier --compilation_level WHITESPACE_ONLY $minifier_wa
 readonly minifier
 readonly minifycmd
 
+minified_sourcemap_cleaner="../tools/sourcemap-root"
+
 # $1 - base file name
 # $2 - output path
 # $3 - optimization level
-# $4 - defines
+# $4 - extra path info to add to minified sourcemap "sourceRoot" property.
```

Comment:
As of 14.0, we dropped the old "define" parameter's functionality fully.  It had been used to define `KeymanBase.__BUILD__`, but this was replaced and superseded by use of `environment.inc.ts`.

So, I simply repurposed its parameter slot - it's used by the UI modules to ensure the UI's source maps line up path-wise with the rest of KMW source mapped code when viewed in-browser.